chromadb/experimental/README.md

```diff
@@ -0,0 +1,8 @@
+# Experimental Features
+
+The features in this directory are experimental. They might break things, and are subject to change.
+We don't recommend using them in production.
+
+Trying these out? We would love your feedback! Join our [discord server](https://discord.gg/MMeYNTmh3x) and let us know what you think.
```

Comment:
i get a `ModuleNotFoundError: No module named 'chromadb.experimental'` out of the box - may want to leave more instructions in README for live patching of experimental features